src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -101,12 +101,12 @@ const Mentor = ({ dispatch, error, successPage }) => {
             </Title>
             <Col span={18} offset={3}>
               <Title level={5} style={{ paddingTop: '2%' }}>
-                Fill out your user Information
+                Fill out your user information
               </Title>
               <Row style={{ padding: '0 0 3% 3%' }}>
                 <Col md={20} xs={24}>
                   <Form.Item
-                    label="First Name"
+                    label="First name"
```

Comment:
Was this a necessary change? (This is an example comment to demonstrate review!)